### PR TITLE
Feature/keep one filter opened

### PR DESCRIPTION
**Pull request by felipeireslan**
#### What problem is this solving?
On `filter-navigator` there is no way to close a filter opening other.

#### How to test it?
You can test this feature [Here](https://filternavigator--dzarm.myvtex.com/blusas)

#### Screenshots or example usage:
Before this feature, on filter-navigator with horizontally layout was possible to open more than one filter keeping one above others - https://prnt.sc/tp8eqp

Now there is a prop that change this behavior and keep only one filter with opened state - https://prnt.sc/tp8hmt

### CODE REVIEW

**Comment by ricardo7634234** [comment]
Hi guys, can you review this?

**Comment by felipeireslan** [comment]
Hi guys, can you review this PR ?

File: react/components/AvailableFilters.js [hunk 1]
```diff
@@ -10,8 +10,11 @@ const AvailableFilters = ({
   preventRouteChange = false,
   initiallyCollapsed = false,
   navigateToFacet,
-}) =>
-  filters.map(filter => {
+  keepOneFilterOpened = false,
+}) => {
+  const [open, setOpen] = useState({})
```
Comment (iaronaraujo) [inline react/components/AvailableFilters.js:15]: Do you think that using `useContext` would work here instead of `useState`?

Comment (klzns) [reply react/components/AvailableFilters.js:15]: I think useState is more appropriate here

**Comment by iaronaraujo** [comment]
Hi @felipeireslan ! Did you get to see my suggestions yet?

**Comment by ricardo7634234** [comment]
Good morning guys, do we have an update? What´s next?

**Comment by iaronaraujo** [comment]
Solved with another PR

**Pull request closed without merging**